### PR TITLE
[FEAT-OKU/36]-Logout Functionality

### PR DESCRIPTION
**Pull request by twentymurial33**
## Change Type
*Put an `x` in all the boxes that apply
- [X] Feature (non-breaking change which adds functionality)

## Linear issue link
https://linear.app/oku-no-kumo/issue/OKU-36/logout-functionality

## Before & after screenshots or video recordings (Before & After)
Screen Shot 2023-07-10 at 6 53 53 PM

Screen Shot 2023-06-30 at 8 52 22 PM

### CODE REVIEW

File: project_app/components/Header.js [hunk 1]
```diff
@@ -1,11 +1,33 @@
...
+  };
+
+  if (isLoggedOut) {
+    return (
+      <Stack sx={{ width: '100%' }} spacing={2}>
+        <Alert severity="success">You have been logged out successfully.</Alert>
+      </Stack>
+    );
+  }
+
```
Comment (idiglove) [inline project_app/components/Header.js:30]: where is the session log out? what if I go to a different page, would I be also logged out there?

Comment (twentymurial33) [reply project_app/components/Header.js]: Added session log out

File: project_app/components/Header.js [hunk 2]
```diff
@@ -1,11 +1,38 @@
...
+  if (isLoggedOut) {
+    return (
+      <SessionContext.Provider value={{ isLoggedOut, handleLogout }}>
+        <Stack sx={{ width: '100%' }} spacing={2}>
+          <Alert severity="success">
+            You have been logged out successfully.
+          </Alert>
+        </Stack>
+      </SessionContext.Provider>
+    );
```
[the remark this answers is not in the document]
Comment (twentymurial33) [reply project_app/components/Header.js]: Logout is now part of the entire app

File: project_app/components/UserLogin.js [hunk 3]
```diff
@@ -10,29 +10,36 @@ import {
...
   const [password, setPassword] = useState('');
   const [isLoggedIn, setIsLoggedIn] = useState(false);
   const [isOpen, setIsOpen] = useState(false);
 
-  const Login = (event) => {
-    event.preventDefault();
-    console.log(username, password);
+  const Login = (e) => {
+    e.preventDefault();
+    console.log(username, email, password);
```
Comment (idiglove) [inline project_app/components/UserLogin.js:25]: remove console log

Comment (twentymurial33) [reply project_app/components/UserLogin.js:25]: removed the console.log

**Review by idiglove — APPROVED** [review]

**Pull request merged**